src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/BeneficialOwnerIndividualDao.java

```diff
@@ -47,6 +48,15 @@ public class BeneficialOwnerIndividualDao {
     @Field("non_legal_firm_members_nature_of_control_types")
     private List<NatureOfControlType> nonLegalFirmMembersNatureOfControlTypes;
 
+    @Field("trust_control_nature_of_control_types")
+    private List<NatureOfControlType> trustNatureOfNatureOfControlTypes;
```

Comment:
a couple of the new fields have a double 'NatureOf' in their names

Reply:
So good I had to say twice, but restrained it back to once now.